app/models/jwt.rb

```diff
@@ -0,0 +1,7 @@
+class Jwt < ApplicationRecord
+  has_one :registration_state
+  has_one :login_state
```

Comment:
Should this be `has_one` or `belongs_to`?  I'm [doing a similar thing in my branch][1] and I'm never sure which ActiveRecord relation to use

[1]: https://github.com/alphagov/govuk-account-manager-prototype/pull/491/files#diff-e01db8e0a184c7a38ebf3b6f548c5cf469f9eff9a9611b89b95e2b6403b408efR48-R49

Reply:
If I swap these round (put `belongs_to :registration_state`) in the Jwt model, then Rails expects a `registration_state_id` column on `Jwt`.  I'd rather keep things the other way round, since the Jwt could be used in both login and registration states.